packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/AddressForm/guestForm.js

```diff
@@ -145,7 +145,7 @@ const GuestForm = props => {
                         })}
                     >
                         <TextInput
-                            autoComplete="off"
+                            autoComplete="Shipping email"
```

Comment:
For internationalisation all strings should be translatable.

Reply:
done @chris-brabender please have a look